configure.py

```diff
@@ -110,7 +121,7 @@ def main(args):
       f"-S{os.path.join(llvm_path, 'llvm')}",
       "-DLLVM_ENABLE_LLD=ON",
```

Comment:
@nicolasvasilache could you remove this since you introduced an option to specify `DLLVM_ENABLE_LLD` ?

Reply:
ah yes, sorry, see #21 
you should also have an invite in  https://github.com/google/iree-llvm-sandbox/invitations so I can add you to reviews
please also send me the github logins of a few folks I should add on your end